mbtb_app/web_portal/mbtb_portal/views/pages/suspend_user_form.ejs

```diff
@@ -0,0 +1,13 @@
+<div>
+    <h2>Suspend User Form</h2>
```

Comment:
7 -> You will receive those ids here in ejs view as `request_ids` and these ids need to get binded with js file. So that we can send it with suspend_reason to the second controller `suspend-user-with-reason` like below.
```
 <script>
        // binding data recieved from sails controller to JS variable to pass it to angular
        var request_ids_ = <%- JSON.stringify(request_ids) %>;
</script>
```